docs/Deploying-Pega-on-AKS.md

```diff
@@ -478,6 +478,8 @@ Configure the following parameters so the pega.yaml Helm chart matches your depl
 
 - Enable Hazelcast client-server model for Pega Platform 8.6 and later.
 
+- Enable stream with external kafka for Pega Platform 8.4 and later.
```

Comment:
as above, although Pega Platform supported it, only 8.8 and later is supporting this in client-managed cloud, so can we just say 8.8? The previous docs didn't support externalization in the same way, right??

Reply:
No that is not true, CMC is not tied to only 8.8, customer may still deploy 8.4 or other versions which supports external kafka.